src/lesson_1/Run.java

```diff
@@ -0,0 +1,174 @@
+package lesson_1;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Scanner;
+
+public class Run {
```

Comment:
try to follow solid principles in your code (ex. separate logic for admin actions and user actions to new related classes and use instances of new classes)